### PR TITLE
Add tasks for databases management

### PR DESCRIPTION
**Pull request by smortex**
Add basic tasks for managing odoo databases.

* [X] List databases
* [X] Delete a database
* [X] Rename a database
* [x] Backup a database
* [x] Restore a database

### CODE REVIEW

**Comment by neomilium** [comment]
@smortex We renamed your tasks in order to have a consistent namespace as we are working on `addons_*` tasks.

**Comment by smortex** [comment]
Close in favor of #6

**Comment by smortex** [comment]
I have some code for database backup & restore, since #6 seems not to be ready yet, reopen to manage databases from this PR and only focus on addons in #6.

File: tasks/database_restore.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+set -e
+
+if [ ! -f "${PT_filename}" ]; then
+	echo "The backup ${PT_filename} MUST exist" >&2
+	exit 1
+fi
+
+if [ -d ~odoo/.local/share/Odoo/filestore/"${PT_name}" ]; then
+	echo "The database ${PT_name} MOST NOT exist" >&2
```
Comment (mtancoigne) [inline tasks/database_restore.sh:11]: Typo: MOST/MUST

Comment (smortex) [reply tasks/database_restore.sh:11]: PIRATE SLANG, ARGGG!

**Pull request merged**